docs/custom-commands/roles.md

```diff
@@ -0,0 +1,14 @@
+---
+id: command-roles
+sidebar_position: 4
+---
+
+# Roles
+
+Roles allow you to set who has access to each command. This can be set to everyone, non-subscriber, subscriber, VIP, Founder, Moderator, and Broadcaster but also can be used with custom roles. 
```

Comment:
Those are still just roles, should just specify it's the roles defined in *another section*

Reply:
When a roles page is made, I will update with a link on how roles work.